### DISCUSSION TITLE
同一个用户头像没有统一

### DISCUSSION THREAD

**Discussion by lanzhiheng** [geeknote-net/feedback#6] [Q&A]
![image](https://user-images.githubusercontent.com/8858472/124553547-70c6b300-de67-11eb-9149-d86bff16129b.png)

两篇文章应该是同一个人写的吧。头像却不一样？ 还有评论区是不是可以考虑跟Ruby China那样上传截图？

**Comment by chloerei** [comment]
目前片段缓存的 key 没有加用户头像的信息，造成了困惑，要改。

评论区这个我考虑下，目前只是一个 textarea，是换成 post 用的编辑器还是单独写个上传图片的按钮，我想一想。

**Reply by lanzhiheng** [reply]
感觉可以把post那个改一下，做成类似Github或者Ruby China这种布局，放过去。

![image](https://user-images.githubusercontent.com/8858472/124558117-d6696e00-de6c-11eb-9111-29e0b5981d64.png)

我在上面写文章体验还是可以的。等会把那篇发上去。有个功能我挺在意的，或许可以考虑加上，现在上传图片的功能需要
1. 点击按钮。2. 选择图片。对于一些程序员来说还是有点不顺手。很多程序员可能跟我一样，是直接截图，然后就到文本框拷贝了，编辑器能够从剪贴板获取图片的数据流并上传返回上传后的URL以Markdown的形式插入，期望也可以做成这样？
